crates/matrix-sdk/src/room/timeline/inner.rs

```diff
@@ -368,12 +370,93 @@ impl<P: ProfileProvider> TimelineInner<P> {
             .await;
         }
     }
+
+    fn update_event_item(&self, index: usize, event_item: EventTimelineItem) {
+        self.items.lock_mut().set_cloned(index, Arc::new(TimelineItem::Event(event_item)))
+    }
 }
 
 impl TimelineInner {
     pub(super) fn room(&self) -> &room::Common {
         &self.profile_provider
     }
+
+    pub(super) async fn fetch_in_reply_to_details(
+        &self,
+        index: usize,
+        mut item: RemoteEventTimelineItem,
+    ) -> Result<RemoteEventTimelineItem> {
```

Comment:
Why does this return `RemoteEventTimelineItem`, is that a leftover from a previous version? The result doesn't seem to be used.

Reply:
It's written in the PR description :wink: :

> `fetch_in_reply_to_details` returns the updated item to be able to fetch other details with the updated item, even if it is unused now.

Reply:
Oops 😄